docs/setup.rst

```diff
@@ -20,19 +44,24 @@ Cloning your Semester.ly fork will create a directory with all of the code requi
 
          git clone https://github.com/[YOUR-USERNAME]/semesterly
 
-Option 1: Set up using Docker
```

Comment:
I would keep both options, just put "Option 1: Set up using Docker (Preferred)" to clarify that Docker is simply better to use in dev environment